liquibase/changeLogs/capture/tables/discreteGroundWater/changeLog.yml

```diff
@@ -119,3 +119,33 @@ databaseChangeLog:
       changes:
         - sql: alter table ${AQTS_SCHEMA_NAME}.discrete_ground_water add column if not exists approval_level_description text;
         - rollback: alter table ${AQTS_SCHEMA_NAME}.discrete_ground_water drop column approval_level_description;
+
+  - changeSet:
+      author: ayan
+      id: "alter.table.${AQTS_SCHEMA_NAME}.discrete_ground_water.add.created_at"
+      preConditions:
+        - onFail: MARK_RAN
+        - onError: HALT
+        - not:
+            - columnExists:
+                schemaName: ${AQTS_SCHEMA_NAME}
+                tableName: discrete_ground_water
+                columnName: created_at
+      changes:
+        - sql: alter table ${AQTS_SCHEMA_NAME}.discrete_ground_water add column if not exists created_at timestamptz not null default now();
+        - rollback: alter table ${AQTS_SCHEMA_NAME}.discrete_ground_water drop column created_at;
```

Comment:
The other columns in capture-db use timestamp, not timestamptz .. for consistency it would be good to use
timestamp. However, the adjustime function which is used quite a bit, essentially casts the time to timezonetz...
I could be convinced that the column should just be timestamptz to to begin with to avoid needing
adjusting.  If this is to just to be used for support/reporting/tracking issue, then having time zone
would be more convenient.

Reply:
@ssoper-usgs @duselmann 
Any thoughts on above? I could probably go either way.

Reply:
This column shouldn't ever be used in any sort of data download/ui display.  As you said it's just for reporting to help us figure out when a column was changed.  So in that regard, we probably don't need the custom time adjustment function (here: https://github.com/usgs/aqts-capture-db/blob/master/liquibase/changeLogs/capture/functions/adjustTimestamp/adjustTimestamp.sql#L7).  That function is there to adjust inconsistent data into a single format for easy downstream use.  

I could see us appreciating having the time zone, so my vote would be to stick with timestamptz.

Reply:
Thanks Sam, I'll agree on sticking with timestamptz